src/functions/template-tags/organizer.php

```diff
@@ -267,16 +267,18 @@ function tribe_get_organizer_link( $post_id = null, $full_link = true, $echo = f
 
 		$org_id = tribe_get_organizer_id( $post_id );
 		if ( class_exists( 'Tribe__Events__Pro__Main' ) && get_post_status( $org_id ) == 'publish' ) {
+
+			$url    = esc_url_raw( get_permalink( $org_id ) );
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 4 spaces

<sub>hash: d434d99842d9e86e6e22adac05ebd110</sub>

Reply:
```suggestion
			$url = esc_url_raw( get_permalink( $org_id ) );
```